### DISCUSSION TITLE
How to transfer variable values from one Ink story to another Ink story? (in Unity C#)

### DISCUSSION THREAD

**Discussion by bigpng** [inkle/ink#870] [Game integration help]
I have the following variable at globals.ink:
````
VAR affinity = ""
````
And the following in Scene1.ink:
````
~ affinity = "Shenzhen"
````
I want to pass `affinity` to _Scene2.ink_ but I only get the variable value from globals.ink. I have tried to set value and get value through External functions, and tried to play with ObserveVariable. Nothing seems to work. And I cannot find any documentation on how to pass variable values to a different Ink story. I would appreciate any help.

**Comment by Selsynn** [comment]
Hi, 
I am not sure what you are trying to do. Is your game only one ink project with multiple file? (variables defined into `globals.ink` and used in all or any of your scene.ink ?)

If that's the case, it's already working as long as you open in inky the main ink file, the one that has the includes (it could include a ink file that has other includes, but it should have the tree include structure included)
I would strongly suggest that you do not open directly the file you want to work on, but always the one having the include (kind of the root one)

If that's not the case, why do you need to pass along the variable? You should be able to do it with getting the value, and setting the value, but it will be complicated, as I do not think ink was created in the idea of having multiple main game working at the same time in the same projects.
